plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeRegisterTableProcedureTest.java

```diff
@@ -307,6 +307,23 @@ public void testRegisterTableWithInvalidParameter()
                 ".*table_location cannot be null or empty.*");
     }
 
+    @Test
+    public void testRegisterUnregisteredTable()
```

Comment:
Do we need this test duplicated from BaseDeltaLakeConnectorSmokeTest.testUnregisterTable()

Reply:
Yes to cover Glue environment.